app/controllers/foreman_inventory_upload/uploads_settings_controller.rb

```diff
@@ -22,5 +23,12 @@ def self.cloud_connector_status
       return nil unless job
       { id: job.id, task: ForemanTasks::Task.where(:id => job.task_id).first }
     end
+
+    def last_successful_inventory_sync_task
+      task = ForemanTasks::Task.where(label: 'InventorySync::Async::InventoryFullSync', result: 'success')
+                        .reorder('ended_at desc').first
```

Comment:
Is the `reorder` necessary? Why use `.where(...).first` instead of `find_by(...)`?

Reply:
`ForemanTasks::Task.reorder('ended_at desc').find_by(...)` should work. The query it generates is identical to `where(...).reorder(...).first` so it doesn't really matter. Nevermind then